doc/generate_sphinx_doc.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+rm -rf build
+sphinx-build -M html . build -w warnings_sphinx_npmx_zephyr.txt
```

Comment:
Previous script `sphinx_build_local.sh` worked ok, this script also works ok for me.
If the reason is to be consistent with npmx - that's fine, otherwise I do not personally see much improvement with this change.
Moreover it is basically a script to run one command, so maybe we should consider to completely get rid of it.

Reply:
The script changed to be the same as in the npmx repository to avoid unknown errors on GitHub container.

Reply:
Fair enough